.github/workflows/ci.yml

```diff
@@ -11,14 +11,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      matrix:
-        java: ['11', '17']
```

Comment:
nit: i'd keep matrix, with just one option.
it's foreseeable that we will soon want to test with eg 19